src/rprblender/engine/image_filter.py

```diff
@@ -23,6 +24,11 @@
 from rprblender.utils.user_settings import get_user_settings
 
 
+# Setting environment variable to enable using FP16 models.
+# In next RIF versions with variable won't be needed and can be removed.
+os.environ['RIF_AI_FP16_ENABLED'] = "1"
```

Comment:
I wonder if we want to support choosing between then.  I'm wondering if FP32 will be more correct now or in future.

Reply:
I asked Benjamin about it, he suggested to use FP16. 
Anyway, I heard new RIF will be available this week with some fixes in ML, let’s wait it.